edit_course/templatetags/editcourse.py

```diff
@@ -34,6 +35,13 @@ def removeurl(model_object, model_name):
     ))
 
 
+@register.simple_tag
+def submittable(exercises, lobject):
```

Comment:
Opinions on this @Mankro ?

Reply:
It is bad and we need to find a better approach. Templates should be able to use a variable like `lobject.is_submittable`. It can't be necessary to build a list of exercise IDs in order to do this.